main.py

```diff
@@ -42,6 +42,7 @@ def main(config):
             logger=config.get("logger", "tensorboard"),
             local_rank=config["local_rank"],
             amp=config.get("amp", False),
+            cpu=config.get("cpu", False),
```

Comment:
This breaks the energy trainer unless cpu is defined there too.

Reply:
Yeah, makes sense! Thanks for adding the commit here!
Based on our chat on slack, I was thinking that you'll make a new PR (once this was merged) :D